Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -810,7 +810,7 @@ void ParseMarvinSmartsLine(RWMol *mol, const std::string &text,
                            unsigned int line) {
   const unsigned int atomNumStart = 10;
   const unsigned int smartsStart = 15;
-  const unsigned int SMA = 7;
+  //const unsigned int SMA = 7;
```

Comment:
might as well just remove this line instead of commenting it out

Reply:
@greglandrum Sure, I was unsure whether to keep or not; I kept it just because there was more Marvin-related commented lines below. I'll remove it then.